{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -1,52 +1,67 @@
 [tool.poetry]
+authors = ["{{ cookiecutter.author_name }}"]
+description = "{{ cookiecutter.description }}"
 name = "{{ cookiecutter.repo_name }}"
 version = "{{ cookiecutter.version }}"
-description = "{{ cookiecutter.description }}"
-authors = ["{{ cookiecutter.author_name }}"]
 
 include = ["{{ cookiecutter.repo_name }}/py.typed"]
 
+[tool.poetry.scripts]
+cli = 'cli:cli'
+
 [tool.poetry.dependencies]
 python = "^3.9"
+
 python-dotenv = ">=0.21.0"
 
 [tool.poetry.dev-dependencies]
```

Comment:
As they are all dev deps, I think about setting all versions to `"*"`

Reply:
Don't do it because of resolve time drastic increase when doing updates. Make them >= and update them now and then to a newer version. 

Python package managers tend to download all matching versions before doing dependency resolution